detector/aws_instance_not_specified_iam_profile.go

```diff
@@ -2,7 +2,15 @@ package detector
 
 import "github.com/wata727/tflint/issue"
 
-func (d *Detector) DetectAwsInstanceNotSpecifiedIamProfile(issues *[]*issue.Issue) {
+type AwsInstanceNotSpecifiedIAMProfileDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateAwsInstanceNotSpecifiedIAMProfileDetector() *AwsInstanceNotSpecifiedIAMProfileDetector {
+	return &AwsInstanceNotSpecifiedIAMProfileDetector{d}
+}
+
+func (d *AwsInstanceNotSpecifiedIAMProfileDetector) Detect(issues *[]*issue.Issue) {
```

Comment:
__[golint]__
* [golint] exported method AwsInstanceNotSpecifiedIAMProfileDetector.Detect should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->